enterprise/keys/dev.pub.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Add a README.md under enterprise/keys to describe what's the purpose and how it's generated